PyInstaller/building/utils.py

```diff
@@ -89,38 +89,40 @@ def _check_guts_toc(attr, old, toc, last_build, pyc=0):
 
 #---
 
-def add_suffix_to_extensions(toc):
+def add_suffix_to_extension(inm, fnm, typ):
```

Comment:
Seeing as you're already rewriting this function, could you give these parameters some proper names? `fnm` I can guess is probably filename, but even reading the code I can't guess what `inm` is short for.

Reply:
This is a rare case when I'd prefer to see them as they are, because they tend to be referenced by those names everywhere in the code.

Reply:
I suppose...

Reply:
> but I still can't guess what the i stands for...

I suspect it comes from "inside", as per this comment in code:
```
        # 'inm'  - relative filename inside a CArchive
```